src/exception/exception-api.ts

```diff
@@ -62,6 +63,13 @@ const validateEntityExceptionRecords: ValidateRecords<EntityExceptionRecord> = r
   return records;
 };
 
+/**
+ * map submitted text from tsv schema column, to camelCase used in exception services
+ * @param record
+ * @returns camelCased schema name
+ */
+const getSchemaEntity = (record: EntityExceptionRecord) => EntityValues[_.camelCase(record.schema)];
```

Comment:
So this is one of the places where TS let's us down. When you get the value out of `EntityValues` TS says that we are guaranteed to get one of the values from the const `EntityValues`... however `_.camelCase(record.schema)` could potentially be a value not available as a property of `EntityValues` (since `record.schema` is just a string`. So the actual return type has to be declares with `| undefined` otherwise we let errors leak in.

This is entirely a consequence of getting a value using `object[key]` type lookup. :(